python/lsst/ts/MTAOS/config_schema.py

```diff
@@ -43,12 +43,61 @@
   instrument:
     description: Type of instrument for optical feedback control (OFC) to use
     type: string
-    enum: [lsst, comcam, sh, cmos]
     default: comcam
 
+  data_path:
+    description: Path to the data butler.
+    type: string
+    default: /project/shared/
+
+  run_name:
+    description: >-
+      Which name to use when running the pipeline task. This defines
+      the location where the data is written in the butler.
+    type: string
+    default: mtaos_wep
+
+  collections:
+    description: Name of the collections where the data is written in the butler.
+    type: string
+    default: refcats,LSSTComCam/raw/all,LSSTComCam/calib
+
+  pipeline_instrument:
+    description: >-
+      A dictionary that maps the name of the instrument to the name used in
+      the pipeline task.
+    anyOf:
+      - type: object
+        properties:
+          comcam:
+            type: string
+          lsstCam:
+            type: string
+          lsstFamCam:
+            type: string
+        additionalProperties: false
+      - type: "null"
+    default: null
+
+  pipeline_n_processes:
+    description: Number of processes to use when running pipeline.
+    type: integer
+    default: 9
```

Comment:
Please consider to use 1 instead of 9 if the resource in NCSA is limited. In addition, what will happen if you need 9 processors but you could only get 6? The pipeline will run with 6 or need to wait to have 9 to do the task?

If the answer is the former, this attribute might be renamed to `max_pipeline_n_processes` to reflect this.

Reply:
I ran with this configuration in on my laptop (which does not have 9 CPUs) and it worked just fine. I imagine this is how may parallel processing it is going to use. In any case, when running at NCSA, summit, etc this will be running on dedicated and powerful nodes.